doc/lua_api.txt

```diff
@@ -6677,12 +6700,33 @@ Used by `minetest.register_node`.
         -- liquid, flowingliquid drawtypes can only wave like liquids.
 
         sounds = {
+            -- Definition of node sounds to be played at various events.
+            -- All fields in this table are optional.
+
             footstep = <SimpleSoundSpec>,
-            dig = <SimpleSoundSpec>,  -- "__group" = group-based sound (default)
+            -- If walkable, played when object walks on it. If node is
+            -- climbable or a liquid, played when object moves through it
+
+            dig = <SimpleSoundSpec> or "__group",
+            -- While digging node.
+            -- If `"__group"`, then the sound will be
+            -- `default_dig_<groupname>`, where `<groupname>` is the
+            -- name of the tool's digging group with the fastest digging time.
+            -- In case of a tie, one of the sounds will be played (but we
+            -- cannot predict which one)
+            -- Default value: `"__group"`
+
             dug = <SimpleSoundSpec>,
+            -- Node was dug
+
             place = <SimpleSoundSpec>,
+            -- Node was placed. Also played after falling
+
             place_failed = <SimpleSoundSpec>,
+            -- When node placement failed
```

Comment:
It might be worth noting that this sound is never played under normal circumstances (the position is invalid (not loaded) or the player tries to place the node into themselves (ok, this might happen)).
See also https://github.com/minetest/minetest/blob/cf7fda00831372d981415f9a830513b2b28fdf25/src/client/game.cpp#L3256-L3259, https://github.com/minetest/minetest/blob/cf7fda00831372d981415f9a830513b2b28fdf25/src/client/game.cpp#L3429-L3436 and https://github.com/minetest/minetest/blob/cf7fda00831372d981415f9a830513b2b28fdf25/src/client/game.cpp#L3425-L3428.

(I wonder why this sound even exists.)